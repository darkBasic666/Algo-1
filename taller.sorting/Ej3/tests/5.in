5
kiq zucmtohrj ufnwdrkeg kiq acolitkv 