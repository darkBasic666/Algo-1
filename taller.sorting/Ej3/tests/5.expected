2.7e-05secs.
4
acolitkv kiq ufnwdrkeg zucmtohrj 